Poolsense multi device

benneb2 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).
-->
## Breaking change
<!--
  If your PR contains a breaking change for existing users, it is important
  to tell them what breaks, how to make it work again and why we did this.
  This piece of text is published with the release notes, so it helps if you
  write it towards our users, not us.
  Note: Remove this section if this PR is NOT a breaking change.
-->
New Device ID feature has been added. The DeviceID is now part of the sensor name. Because the name changed, the sensor will need to be deleted and then added again. 

## Proposed change
<!--
  Describe the big picture of your changes here to communicate to the
  maintainers why we should accept this pull request. If it fixes a bug
  or resolves a feature request, be sure to link to that issue in the
  additional information section.
-->
This change allows users to add multiple poolsense devices under a single user.

## Type of change
<!--
  What type of change does your PR introduce to Home Assistant?
  NOTE: Please, check only 1! box!
  If your PR requires multiple boxes to be checked, you'll most likely need to
  split it into multiple PRs. This makes things easier and faster to code review.
-->

- [x] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [x] New feature (which adds functionality to an existing integration)
- [x] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information
<!--
  Details are important, and help maintainers processing your PR.
  Please be sure to fill out additional details, if applicable.
-->

- This PR fixes or closes issue: fixes #
https://github.com/haemishkyd/poolsense/issues/1
- This PR is related to issue: 
https://github.com/haemishkyd/poolsense/pull/3
- Link to documentation pull request: 

## Checklist
<!--
  Put an `x` in the boxes that apply. You can also fill these out after
  creating the PR. If you're unsure about any of them, don't hesitate to ask.
  We're here to help! This is simply a reminder of what we are going to look
  for before merging your code.
-->

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] I have followed the [development checklist][dev-checklist]
- [x] The code has been formatted using Black (`black --fast homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [x] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [x] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [x] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description.
- [ ] Untested files have been added to `.coveragerc`.

The integration reached or maintains the following [Integration Quality Scale][quality-scale]:
<!--
  The Integration Quality Scale scores an integration on the code quality
  and user experience. Each level of the quality scale consists of a list
  of requirements. We highly recommend getting your integration scored!
-->

- [ ] No score or internal
- [ ] 🥈 Silver
- [ ] 🥇 Gold
- [ ] 🏆 Platinum

<!--
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
-->

To help with the load of incoming pull requests:

- [ ] I have reviewed two other [open pull requests][prs] in this repository.

[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone

<!--
  Thank you for contributing <3

  Below, some useful links you could explore:
-->
[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html
[quality-scale]: https://developers.home-assistant.io/docs/en/next/integration_quality_scale_index.html
[docs-repository]: https://github.com/home-assistant/home-assistant.io

benneb2 committed:
Add multi device for poolsense integration

benneb2 committed:
Update poolsense library to version 0.1.0

homeassistant commented:
Hello @benneb2,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`15c6f69a478e9f74b10edb2e5c14237144784960`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;15c6f69a478e9f74b10edb2e5c14237144784960) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.
- [`460dadcc30c2cd0d4b7881c0c401c75d0ee47372`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;460dadcc30c2cd0d4b7881c0c401c75d0ee47372) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hello @benneb2,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`a41acfbfdfd40c848834a7181aab2fe938b658ed`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;a41acfbfdfd40c848834a7181aab2fe938b658ed) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.
- [`15c6f69a478e9f74b10edb2e5c14237144784960`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;15c6f69a478e9f74b10edb2e5c14237144784960) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.
- [`460dadcc30c2cd0d4b7881c0c401c75d0ee47372`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;460dadcc30c2cd0d4b7881c0c401c75d0ee47372) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hello @benneb2,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`460dadcc30c2cd0d4b7881c0c401c75d0ee47372`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;460dadcc30c2cd0d4b7881c0c401c75d0ee47372) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hello @benneb2,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`460dadcc30c2cd0d4b7881c0c401c75d0ee47372`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;core&#x2F;pull&#x2F;56194&#x2F;commits&#x2F;460dadcc30c2cd0d4b7881c0c401c75d0ee47372) - This commit has something that looks like an email address (bernhardheesen@Bernhards-MacBook-Pro.local). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

homeassistant commented:
Hi @benneb2, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;core#56194).

Once you do that we will be able to review and accept this pull request.

Thanks!

bdraco reviewed (changes requested):
Please fix conflicts

haemishkyd commented:
I might be missing something but reviewing the commits I cannot see any conflicts. Please point me to where these conflicts are.
Apologies if this is a stupid question.

frenck committed:
Bumped version to 2021.10.0b0

raman325 committed:
Add strings for new zwave_js config flow keys (#56844)

milanmeu committed:
Upgrade aionanoleaf to 0.0.2 (#56845)

emontnemery committed:
Correct database migration to schema version 22 (#56848)

balloob committed:
Bumped version to 2021.10.0b1

Danielhiversen committed:
Opengarage bug fix (#56869)

* Opengarage bug fix

Signed-off-by: Daniel Hjelseth Høyer <github@dahoiv.net>

* Opengarage bug fix

Signed-off-by: Daniel Hjelseth Høyer <github@dahoiv.net>

* Deprecated open garage config

Signed-off-by: Daniel Hjelseth Høyer <github@dahoiv.net>

* Deprecated open garage config

Signed-off-by: Daniel Hjelseth Høyer <github@dahoiv.net>

cdce8p committed:
Fix check_control_message short description (#56876)

OttoWinter committed:
Bump aioesphomeapi from 9.1.0 to 9.1.2 (#56879)

chemelli74 committed:
Revert fritz pref_disable_new_entities handling (#56891)

emontnemery committed:
Adjust state class of solarlog yield and consumption sensors (#56824)

Kane610 committed:
Use native unit of measurement in deCONZ sensors (#56897)

Kane610 committed:
CLIPGenericFlag should be deCONZ sensor not binary sensor (#56901)

RicArch97 committed:
Handle missing serial extended parameters in crownstone (#56864)

cdce8p committed:
Fix bmw_connected_drive battery icon (#56884)

balloob committed:
Bumped version to 2021.10.0b2

crazyfx1 committed:
Fix vicare binary sensor (#56912)

balloob committed:
Bump netdisco to 3.0.0 (#56903)

bieniu committed:
Fix `Unable to serialize to JSON` error in Xiaomi Miio (#56929)

MatthewFlamm committed:
Bump pynws: fix unit code bug (#56923)

bdraco committed:
Add DHCP support for TPLink KL430, KP115 (#56932)

bachya committed:
Address beta review comments for WattTime (#56919)

bdraco committed:
Bump PyFlume to 0.6.5 to fix compat with new JWT (#56936)

Changelog: https://github.com/ChrisMandich/PyFlume/compare/5476fd67cfc8be768c0ea810d248f39399e038d1...v0.6.5

Commit:
[ci skip] Translation update

chemelli74 committed:
Add sleep_period to log for easier debugging (#56949)

RenierM26 committed:
Fix Switchbot unsupported SB types (#56928)

mib1185 committed:
Set unique id while SSDP discovery of Synology DSM (#56914)

fredrike committed:
Update pypoint to use v5 of backend API (#56934)

bdraco committed:
Add dhcp discovery for TPLink EP10 (#56955)

balloob committed:
Bumped version to 2021.10.0b3

bramkragten committed:
Update frontend to 20211002.0 (#56963)

Commit:
[ci skip] Translation update

zlinoliver committed:
Fix Tuya v2 fan percentage (#56954)

* fix:Some fans do not have a fan_speed_percent key

* fix comment format issue

Co-authored-by: erchuan <jie.zheng@tuya.com>

bachya committed:
Fix incorrect handling of hass.data in WattTime setup (#56971)

bdraco committed:
Fix yeelight state when controlled outside of Home Assistant (#56964)

thecode committed:
Bump aioshelly to 1.0.2 (#56980)

chishm committed:
Disable discovery for dlna_dmr until it is more selective (#56950)

StevenLooman committed:
Bump async-upnp-client to 0.22.5 (#56989)

StevenLooman committed:
Fix upnp invalid key in ssdp discovery_info (#56986)

bdraco committed:
Round tplink energy sensors to prevent insignificant updates (#56999)

- These sensors wobble quite a bit and the precision did
  not have sensible limits which generated a massive amount
  of data in the database which was not very useful

cgtobi committed:
Bump pyatmo to v6.1.0 (#57014)

raman325 committed:
Bump zwave-js-server-python to 0.31.2 (#57007)

bdraco committed:
Bump yeelight to 0.7.6 (#57009)

- Fixes compat with Lamp15 model

- May improvment Monob model drops seen in #56646

Changes: https://gitlab.com/stavros/python-yeelight/-/commit/0b94e5214e3375f20defa386067ef6cb058c872c

cgtobi committed:
Fix camera tests (#57020)

Commit:
[ci skip] Translation update

bdraco committed:
Add DHCP support for TPLink KP400 (#57023)

jjlawren committed:
Shorten album titles when browsing artist (#57027)

filcole committed:
Use pycarwings2.12 for Nissan Leaf integration (#56996)

oncleben31 committed:
Meteofrance fix #56975 (#57016)

dgomes committed:
Ignore utility_meter restore state if state is invalid (#57010)

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

balloob committed:
Bumped version to 2021.10.0b4

balloob committed:
Mark auth voluptuous schema fields as required (#57003)

zlinoliver committed:
Fix Tuya v2 login issue (#56973)

* fix login issue

* fix:login error

* update COUNTRY_CODE_CHINA line location

* added one blank line

* feat:added line #L88 was not covered by tests

* ci build errors

Co-authored-by: erchuan <jie.zheng@tuya.com>

OttoWinter committed:
ESPHome fix zeroconf add_listener issue (#57031)

OttoWinter committed:
Bump aioesphomeapi from 9.1.2 to 9.1.4 (#57036)

emontnemery committed:
Prevent opening of sockets in watttime tests (#57040)

koying committed:
Universal media player: consider unknown as inactive child state (#57029)

emontnemery committed:
Evict purged states from recorder's old_state cache (#56877)

Co-authored-by: J. Nick Koston <nick@koston.org>

ludeeus committed:
Rewrite tuya config flow (#57043)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>
Co-authored-by: Franck Nijhof <git@frenck.dev>

balloob committed:
Bumped version to 2021.10.0b5

balloob committed:
[ci skip] Translation update

emontnemery committed:
Improve sensor statistics validation (#56892)

bdraco committed:
Update esphome reconnect logic to use newer RecordUpdateListener logic (#57057)

thecode committed:
Fix: Shelly Gen2 - filter unsupported sensors (#57065)

balloob committed:
Update Tuya code owners (#57078)

Commit:
[ci skip] Translation update

bramkragten committed:
Update frontend to 20211004.0 (#57073)

balloob committed:
Fix energy gas price validation (#57075)

balloob committed:
Bumped version to 2021.10.0b6

frenck committed:
Prevent Tuya from accidentally logging credentials in debug mode (#57100)

frenck committed:
Remove Python shebang line from Tuya integration files (#57103)

OttoWinter committed:
Bump aioesphomeapi from 9.1.4 to 9.1.5 (#57106)

frenck committed:
Small code styling tweaks for Tuya (#57102)

frenck committed:
Bump tuya-iot-py-sdk to 0.5.0 (#57110)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

jrester committed:
Update tesla_powerwall to 0.3.11 (#57112)

balloob committed:
Bump aiohue to 2.6.3 (#57125)

indykoning committed:
Fix Growatt login invalid auth response (#57071)

balloob committed:
Reinstate asking for country in Tuya flow (#57142)

raman325 committed:
Bump zwave-js-server-python to 0.31.3 (#57143)

bdraco committed:
Fix yeelight connection when bulb stops responding to SSDP (#57138)

balloob committed:
Bumped version to 2021.10.0b7

chemelli74 committed:
Fix SamsungTV shutdown race condition (#57149)

chemelli74 committed:
Fix Fritz shutdown race condition (#57148)

mindmelting committed:
Updated amberelectic attributes to reflect unit change to $/kWh (#57109)

balloob committed:
Guard upnp create device (#57156)

balloob committed:
Bumped version to 2021.10.0b8

balloob committed:
Bumped version to 2021.10.0b9

fredrike committed:
Update Daikin config_flow with better error handling (#57069)

tschamm committed:
Skip link local addresses in bosch_shc discovery step (#57074)

bramkragten committed:
Update frontend to 20211006.0 (#57164)

jyavenard committed:
Change energy state class to STATE_CLASS_TOTAL (#56974)

emontnemery committed:
Correct migration to recorder schema 18 (#57165)

starkillerOG committed:
Remove Netgear tracker link_rate check on Orbi (#57032)

* Netgear tracker: remove link_rate check on Orbi

* fix debug message

* Add orbi models

* check start of model in V2 check

* fix black

frenck committed:
Bumped version to 2021.10.0

frenck committed:
Merge pull request #57179 from home-assistant/rc

MartinHjelmare committed:
Fix netgear config flow import (#57253)

emontnemery committed:
Validate initial value for input_datetime (#57256)

bieniu committed:
Update led brightness select state only if valid data is available, Xiaomi Miio integration (#57197)

* Update state if there is valid data

* Add comment

bdraco committed:
Discover tplink devices periodically (#57221)

- These devices sometimes do not respond on the first try
  or may be subject to transient broadcast failures, or
  overloads. We now try discovery periodically once the
  integration has been loaded.

- We used to try this 4x at startup, but that solution
  seemed to aggressive as we want to be sure we pickup
  the devices after startup as well since the network
  will likely be more calm after startup.

Danielhiversen committed:
Bump Mill library to 0.6.1 (#57261)

bdraco committed:
Fix RGB only (no color temp) devices with tplink (#57267)

bramkragten committed:
Update frontend to 20211007.0 (#57268)

raman325 committed:
Convert val to str when needed while calling zwave_js.set_value (#57216)

emontnemery committed:
Correct SQL query generated by get_metadata_with_session (#57225)

Co-authored-by: Franck Nijhof <git@frenck.dev>

rytilahti committed:
Fix transition handling for tplink lights (#57272)

* Fix transition handling for tplink light

* Apply suggestions from code review

* Test that all transitions are passed correctly

* Fix linting

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

bdraco committed:
Bump yeelight to 0.7.7 (#57290)

bdraco committed:
Bump HAP-python to 4.30 (#57284)

rytilahti committed:
Stopgap fix for inconsistent upstream API of tplink dimmers (#57285)

bdraco committed:
Recreate the powerwall session/object when attempting relogin (#56935)

balloob committed:
Bumped version to 2021.10.1

balloob committed:
Merge pull request #57294 from home-assistant/rc

StevenLooman committed:
Fix multiple upnp/ssdp issues (#57314)

bdraco committed:
Migrate tplink hosts that were previously imported from yaml (#57308)

milanmeu committed:
Upgrade aionanoleaf to 0.0.3 to fix deadlock (#57312)

milanmeu committed:
Fix Nanoleaf light turn_off transition (#57305)

starkillerOG committed:
Netgear fix port and device model beeing overwritten (#57277)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

emontnemery committed:
Improve state of cover groups (#57313)

Co-authored-by: Paulus Schoutsen <balloob@gmail.com>

balloob committed:
Guard for bad last reset (#57344)

balloob committed:
Bumped version to 2021.10.2

balloob committed:
Merge pull request #57355 from home-assistant/rc

thecode committed:
Fix Shelly RGB/W supported color mode detection (#57359)

RenierM26 committed:
Bump Switchbot library (#57367)

* Bump dependency to fix SB password problem.

* Bump API version.

tkdrob committed:
Catch errors for efergy (#57326)

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

bramkragten committed:
Update frontend to 20211007.1 (#57385)

MatthewFlamm committed:
Bump pynws to 1.3.2 (#57361)

ludeeus committed:
Bump pyhaversion to 21.10.0 (#57377)

mvn23 committed:
Fix `opentherm_gw.set_clock` `time` parameter name (#57398)

mvn23 committed:
Fix default parameter values for service opentherm_gw.set_clock (#57397)

bdraco committed:
Do all of dhcp scapy startup in the executor (#57392)

bdraco committed:
Remove executor calls in isy994 as its fully async (#57394)

bdraco committed:
Restore yeelight workaround for failing to update state after on/off (#57400)

bdraco committed:
Use switch format unique ids for tplink dimmers (#57346)

thecode committed:
Fix Shelly button filter empty event (#57427)

thecode committed:
Fix Shelly button type in roller mode (#57429)

bdraco committed:
Bump zeroconf to 0.36.8 (#57451)

- Changelog: https://github.com/jstasiak/python-zeroconf/releases/tag/0.36.8

thecode committed:
Fix netgear renamed mdi icons (#57431)

balloob committed:
Bumped version to 2021.10.3

balloob committed:
Merge pull request #57455 from home-assistant/rc

StevenLooman committed:
Fix upnp creating derived sensors (#57436)

gjong committed:
Upgrade youless library to fix missing sensor LS110 (#57366)

micha91 committed:
Upgrade aiomusiccast to tolererate not decodable characters (#57461)

starkillerOG committed:
Fix Netgear orbi port in ssdp discovery (#57432)

tkdrob committed:
Fix referenced before assignment in modem_callerid (#57460)

chpego committed:
Bump youtube-dl to 2021.06.06 (#57490)

bdraco committed:
Simplify yeelight setup to improve reliability (#57500)

emontnemery committed:
Detect if mysql and sqlite support row_number (#57475)

balloob committed:
Bumped version to 2021.10.4

frenck committed:
Merge pull request #57517 from home-assistant/rc

Danielhiversen committed:
Bump Mill library to 0.6.2 (#57533)

Z1ni committed:
Fix Fast.com autoupdate (#57552)

bachya committed:
Bump simplisafe-python to 11.0.7 (#57573)

emontnemery committed:
Correct detection of row_number support for MariaDB (#57663)

bachya committed:
Ensure Notion device name is stored as a string (#57670)

peternijssen committed:
Bump spiderpy to 1.4.3 (#57675)

thecode committed:
Fix Shelly humidity sensor available condition (#57721)

gjong committed:
Fix Youless state class for power total sensor (#57758)

bdraco committed:
Reconnect and retry yeelight commands after previous wifi drop out (#57741)

bdraco committed:
Always send color/temp when switching from an effect in yeelight (#57745)

chishm committed:
Log reason for DLNA-DMR device becoming unavailable (#57516)

balloob committed:
Bumped version to 2021.10.5

StevenLooman committed:
Fix broken upnp derived sensors reporting b/s instead of kb/s (#57681)

balloob committed:
Merge pull request #57793 from home-assistant/rc

bdraco committed:
Bump bond-api to 0.1.14 (#57874)

squio committed:
Fix device class for energy plugwise sensors (#57803)

Co-authored-by: Franck Nijhof <git@frenck.dev>

cdce8p committed:
Revert "Fix bmw_conntected_drive check_control_message short description" (#57928)

This reverts commit acda3afe63fd051d035192a3983d2d48ff35289c.

bdraco committed:
Prevent yeelight discovery from overloading the bulb (#57820)

bdraco committed:
Avoid setting up harmony websocket from discovery (#57589)

bachya committed:
Fix bug that prevents multiple instances of Tile (#57942)

balloob committed:
Bumped version to 2021.10.6

balloob committed:
Merge pull request #57944 from home-assistant/rc

benneb2 committed:
chore: merge master

mib1185 reviewed (changes requested):
@benneb2 you first should rebase your branch onto the current `dev`

frenck commented:
Closing this PR, as it hasn't been addressed/activity on it since September. Please feel free to re-open a PR when ready to work on it again. Thanks 👍
